src/db/spanner/support.rs

```diff
@@ -79,7 +54,6 @@ pub enum SpannerType {
     Struct,
 }
 
-#[cfg(feature = "google_grpc")]
 impl Into<googleapis_raw::spanner::v1::type_pb::Type> for SpannerType {
```

Comment:
Not terribly important but we can also optionally kill the SpannerType enum. It's only for supporting both spanner libs simultaneously.

Reply:
Looks like this is a bigger change, so I opened a new bug for it:  #261.